simphony/io/h5_cuds.py

```diff
@@ -152,6 +155,47 @@ def add_mesh(self, mesh):
         self._file.flush()
         return m
 
+    def add_lattice(self, lattice, record=None):
```

Comment:
LB modeling engines internally store (at least) DENSITY, VELOCITY, and DISTRIBUTION (the above mentioned 19-component vector). On the other hand, at the SimPhoNy side, lattices typically need to store/hold only, for example, DENSITY and VELOCITY (usual variables of interest). So, if H5Cuds currently allows storing a subset of CUBA keywords, this case is already ok from my point of view.

However, when we proceed to more advanced couplings between methods and scales, it might become necessary/fruitful to bring the DISTRIBUTION data (the true/intrinsic variable of LBM) over to the SimPhoNy side. But even in this case, it is probably not necessary to store the distributions to a file (at least when saving intermediate time snapshots of the lattice state). Hence, a demand for storing a subset of a subset of CUBA keywords can arise.

At the moment I don't know how such cases would be best handled. But at the current development stage, and in order to proceed, we can probably leave this feature to future discussion and design.